api.md

```diff
@@ -412,6 +413,23 @@ queries.
 
 ---
 
+## CREATE INDEX [](create_index)
+
+```SQL
+CREATE INDEX ON <hypertable> WITH (timescaledb.transaction_per_chunk);
+```
+
+This functions much like a regular `CREATE INDEX` except that it uses
+a separate transaction for each chunk it creates an index on, instead
+of using a single transaction for the entire hypertable. This allows
+`INSERT`s, and other operations to to be performed concurrently during
+most of the duration of the `CREATE INDEX`.
+
+>:TIP: This version of `CREATE INDEX` can be used as an alternative to `CREATE INDEX CONCURRENTLY`
```

Comment:
Can we add a sentence here on how the two are different?

Reply:
@dianasaur323 hmmm, the most important one is that `CREATE INDEX CONCURRENTLY` doesn't work on hypertables at the moment.

Reply:
If we do implement it, I think the difference would end up being that this version blocks INSERTs to the last chunk for a slightly longer time
(honestly, I do wonder if we should have just had `CREATE INDEX CONCURRENTLY` call this)

Reply:
Had some more discussion with @davidkohn88 about this: since we don't implement `CREATE INDEX CONCURRENTLY` on hypertables at the moment, a users options are only: use regular `CREATE INDEX`, use this, or create indexes on chunks manually. As such, the main use for this tip to point users searching for how to run `CREATE INDEX CONCURRENTLY` on a hypertable at this instead. As such, I don't think a comparison with `CREATE INDEX CONCURRENTLY` is really relevant here, and is more suited for the blog post.

Reply:
As an aside, should we add a hint to the error from `CREATE INDEX CONCURRENTLY` that says use `CREATE INDEX ... WITH (timescaledb. transaction_per_chunk)` instead?

Reply:
The little note you put in there in the new version is perfect!

Reply:
Adding a note might be good for UX, but I don't feel particularly strongly about it. Think that's a nice clean-up UX feature.